### PR TITLE
Have notebooks refer to where they are hosted

### PR DESCRIPTION
**Pull request by adamnsch**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [x] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [x] Your contribution is covered by tests

### CODE REVIEW

File: doc/modules/ROOT/pages/tutorials/fastrp-and-knn.adoc [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 = Product recommendations with kNN based on FastRP embeddings
 
+This Jupyter notebook is hosted
+https://github.com/neo4j/graph-data-science-client/blob/main/examples/fastrp-and-knn.ipynb[here]
+in the Neo4j Graph Data Science Client Github repository.
+
```
Comment (Mats-SX) [inline doc/modules/ROOT/pages/tutorials/fastrp-and-knn.adoc:8]: Did you write this manually, or got this part through the generator?

File: doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 = Heterogeneous Node Classification with HashGNN and Autotuning
 
+This Jupyter notebook is hosted
+https://github.com/neo4j/graph-data-science-client/blob/main/examples/heterogeneous-node-classification-with-hashgnn.ipynb[here]
+in the Neo4j Graph Data Science Client Github repository.
+
```
Comment (Mats-SX) [inline doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:8]: These appear now above the `open in colab` button. I prefer having the button on top.

Comment (Mats-SX) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:8]: Screenshot 2023-02-28 at 10 52 23

Comment (Mats-SX) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:8]: Can we move it down one cell perhaps? And update instructions?

**Review by Mats-SX — COMMENTED** [review]
Not blocking, but in my opinion we should let the `open in colab` button be at the very top

File: doc/modules/ROOT/pages/tutorials/fastrp-and-knn.adoc (hunk 1 above)
Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/fastrp-and-knn.adoc:8]: This goes through the generator

File: doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc (hunk 2 above)
Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:8]: Ok :) I prefer it slightly the other way (why I made it so) but don't have a strong opinion. So I'll move it then

Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc]: Done!

**Pull request merged**